src/post/post.entity.ts

```diff
@@ -36,6 +43,7 @@ export interface PostData {
     publishedAt?: Date;
     inReplyTo?: Post | null;
     apId?: URL | null;
+    attachments?: PostAttachment[] | null;
```

Comment:
🤔 Do we need `null` for this? Like could it not always just be `PostAttachment[]` and if there are no attachments its an empty array? Then when we need to check if there are attachments we just do `attachments.length > 0`? Would mean we don't have to explicitly check if the value is `null` before trying to check `length`

Reply:
Good point. I'll update it!